backend/portfolio42_api/models.py

```diff
@@ -35,6 +35,9 @@ class Cursus(models.Model):
 	kind = models.CharField(max_length=50)
 	updated_at = models.DateTimeField(auto_created=True, auto_now=True)
 
+	skills = models.ManyToManyField('Skill', through='CursusSkill')
+	users = models.ManyToManyField('User', through='CursusUser')
+
```

Comment:
Here we should add projects which has a relationship Many to Many with the cursus.

Reply:
So basically add manytomany on both parts of the relation?

Reply:
Yes, when need access from both sides.
cursusObj1.projects => returns list of projects
projectObj1.cursus => returns list of cursus